srcs/responses/DeleteResponse.cpp

```diff
@@ -19,39 +20,77 @@ DeleteResponse& DeleteResponse::operator=(const DeleteResponse& rhs) {
 }
 
 // Public methods
+// void    DeleteResponse::executeDeleteResponse(Request& req) {
+//     std::ofstream   tmpFile;
+//     const char*     file_path = "tmp/delete.txt";
+//     std::string     command;
+//     // int             ret;
+//     
+//     // debugging check
+//     std::cout << "raw DELETE body is " << req.getRawBody() << std::endl;
+//     
+//     // write the body of the POST request to a tmp file
+//     tmpFile.open(file_path);
+//     tmpFile << req.getRawBody();
+//     tmpFile.close();
+//     
+//     // execute the php script with the contents of the file
+//     command = "php " + _resource + " " + file_path; 
+//     std::cout << "full command to be exec is: " << command << std::endl;
+//     std::system(command.c_str());
+//
+//     // delete the tmp file
+//     remove(file_path);
+//     // TODO: check return value ret and throw error if fail...
+// }
+
+std::string getResourceID(std::string resource) {
+    std::string numberStr;
+    
+    std::string::size_type pos = resource.rfind('/');
+    if (pos != std::string::npos) {
+        ++pos;
+    }
+    std::string::size_type endPos = resource.find(".jpeg", pos);
+    if (endPos != std::string::npos) {
+    numberStr = resource.substr(pos, endPos - pos);
+    }
+    return numberStr;
+}
+
 void    DeleteResponse::executeDeleteResponse(Request& req) {
     std::ofstream   tmpFile;
-    const char*     file_path = "tmp/delete.txt";
+    std::string     file_path = "tmp/delete-entry.txt";
     std::string     command;
-    // int             ret;
-    
-    // debugging check
-    std::cout << "raw DELETE body is " << req.getRawBody() << std::endl;
-    
+    std::string     resource_id; 
+
+    // get resource id
+    (void)req;
+    resource_id = getResourceID(_resource);
+
     // write the body of the POST request to a tmp file
```

Comment:
Why writing to a file instead of a variable?

Reply:
Because I can easily pass files to php in the following piece of code:
`    command = "php cgi-bin/delete-entry.php " + file_path; `

There is probably other ways to do it but that's how I did it for post requests so I'm also doing the same way for DELETE requests